### ISSUE TITLE
Static voice chat

### ISSUE THREAD

**Issue by Chicub** [ottomated/CrewLink#253]
Whenever my friends talk in game they are sound like static but to them it’s fine.

**Comment by Latif-99** [comment]
same thing for me

**Comment by gbrldz** [comment]
+1 for me. Both my DAC and front panel case jack.

**Comment by TheSpringBoy** [comment]
Hi.
solution in #250 
Just switch voice server abd you ahould be fine

**Comment by gbrldz** [comment]
I did. Issue persists.

**Comment by TheSpringBoy** [comment]
Hmmmm
Make sure you select the correct speaker

**Comment by gbrldz** [comment]
Yeah, I tried all the options in the dropdown menu. All the same sadly

**Comment by TheSpringBoy** [comment]
The only thing I can still think about is making sure windows is up to date, maybe there is some driver out of date

**Comment by gbrldz** [comment]
As far as I know, everything is up to date. I can live with it for now. Thanks for trying to help!

**Comment by duckyknights** [comment]
Same issue for me, changed server, created my own voice server, run updates on Windows and Among Us etc, still getting static when people speak. No one else hears it on our server.

**Comment by Latif-99** [comment]
Found the fix. Go to your sound control panel on windows and click your playback device and clicked advanced, then change the volume format to something lower. Mine sounds good at 96000 Hz

**Comment by jeepee12** [comment]
You need to restart it after changing the settings.

**Comment by kYwzor** [comment]
> Found the fix. Go to your sound control panel on windows and click your playback device and clicked advanced, then change the volume format to something lower. Mine sounds good at 96000 Hz

I can confirm this fixed the issue for me.

**Comment by MiLkTiddiE** [comment]
After the new version came out im now experiencing the static issue and I cant find how to solve it in the new menu

**Comment by ybadragon** [comment]
This is also happening with a few people in my play group, something to look into, everyone who is having the issue in my playgroup all have Razer Headsets and the static is significantly worse when using the THX Spatial - Synapse option for Speaker
